src/JuMP.jl

```diff
@@ -467,14 +467,19 @@ function Base.setindex!(m::JuMP.Model, value, name::Symbol)
 end
 
 # usage warnings
+# If extensions do not overwrite this, the warning won't be shown
+function _operator_warn(::AbstractVariableRef) end
```

Comment:
This design needs more thought and documentation if we're going to make it part of the extension API. The starting underscore should be avoided for extension APIs. When exactly is this function called, what should an overwrite look like, etc...

Reply:
Done :)